changelog/unreleased/enhancement-add-editors-to-application-menu

```diff
@@ -0,0 +1,10 @@
+Enhancement: Add editors to the application menu
+
+We've added 'text-editor' and 'draw.io' to the application menu, so the user can easily
+open those apps with a one-click approach.
+We've also fixed a bug, where the full path of a resource was missing while hovering it, for example in
```

Comment:
Completely unrelated, why not add the bugfix as a second changelog item with `bugfix` prefix? Doesn't do any harm that the two changelog items have the same PR url.